requirements_for_test.txt

```diff
@@ -1,9 +1,9 @@
 -r requirements.txt
-pytest==5.4.3
-pytest-mock==1.13.0
-pytest-cov==2.10.1
-pytest-xdist==1.34.0
+pytest==6.2.3
+pytest-mock==3.5.1
+pytest-cov==2.11.1
+pytest-xdist==2.2.1
 requests-mock==1.8.0
-freezegun==0.3.15
-flake8==3.8.4
-flake8-print==3.1.4
+freezegun==1.1.0
+flake8==3.9.1
```

Comment:
are there any conflicts with notification-utils and some of the other projects that depend upon this project? I'm seeing some differences between this and notification-api and notification-admin, i.e flake=3.8.4 within -api and -admin but is getting updated within -utils

Reply:
The plan is to bump utils first, pull the new version on admin and API and update dependencies there as well